From: Kill Your TV <killyourtv@i2pmail.org>
Date: Sat, 20 Dec 2014 03:12:18 +0000
Subject: cxx flag

---
 qtox.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qtox.pro b/qtox.pro
index 98323e4..fab3f32 100644
--- a/qtox.pro
+++ b/qtox.pro
@@ -36,7 +36,7 @@ FORMS    += \
     src/widget/form/setpassworddialog.ui \
     src/widget/form/settings/advancedsettings.ui
     
-CONFIG   += c++11
+CONFIG   += gnu++11
 
 # Rules for creating/updating {ts|qm}-files
 include(translations/i18n.pri)
